src/main/java/software/amazon/payloadoffloading/ServerSideEncryptionFactory.java

```diff
@@ -0,0 +1,11 @@
+package software.amazon.payloadoffloading;
+
+public class ServerSideEncryptionFactory {
```

Comment:
Can we add unit tests for the newly added `SseStrategy` implementations?

Reply:
done